### PR TITLE
Add Interia

### PR DESCRIPTION
**Pull request by mimi89999**

### CODE REVIEW

**Review by ApCoder123 — CHANGES_REQUESTED** [review]
Hi there,

If a website does not support 2FA, the `contact` field is necessary. Please see the [guidance for adding sites without 2FA](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#adding-a-site-that-does-not-support-tfa). 

If a website does support 2FA, the `tfa` field is necessary. Please see the [guidance for adding sites with 2FA](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#adding-a-site-that-supports-tfa)

Please add one of these fields depending on the site's support for 2FA.

File: img/i/interia.pl.svg [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+   fill="#090947"
+   id="path2" />
+</svg>
```
Comment (ApCoder123) [inline img/i/interia.pl.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="189.85" height="47.482" viewBox="0 0 189.849 47.482"><path d="M0 47h10.457V14.236S6.319 10.334 0 8.78V47zm131.76 0h10.457V8.78c-6.318 1.554-10.457 5.456-10.457 5.456V47zM28.75 8.161c-8.707.307-14.807 6.075-14.807 6.075V47H24.4V21.258c.407-1.65 2.762-2.758 4.546-3.145 3.82-.83 8.002 1.453 8.002 5.504V47h10.457V23.109c0-11.022-9.935-15.256-18.655-14.948zm155.56 17.63c-1.008-10.076-9.422-17.675-19.572-17.675-10.847 0-19.672 8.831-19.672 19.686 0 10.853 8.825 19.681 19.672 19.681 5.119 0 9.955-1.985 13.679-5.601L179.76 47h10.088l-5.538-21.209zm-19.573 11.421c-5.185 0-9.398-4.216-9.398-9.396 0-5.183 4.214-9.398 9.398-9.398 5.182 0 9.397 4.216 9.397 9.398.002 5.18-4.215 9.396-9.397 9.396zM88.325 8.206c-10.805 0-19.594 8.789-19.594 19.594 0 10.803 8.79 19.592 19.594 19.592 8.108 0 15.266-4.877 18.235-12.426l.16-.408H95.425l-.088.09a9.726 9.726 0 0 1-7.012 2.955c-3.885 0-7.441-2.348-8.987-5.887h28.185l.047-.242a19.95 19.95 0 0 0 .349-3.676c-.001-10.803-8.79-19.592-19.594-19.592zm-8.839 15.347a9.793 9.793 0 0 1 8.839-5.558 9.788 9.788 0 0 1 8.838 5.558H79.486zm46.163-15.392c-8.709.307-14.808 6.075-14.808 6.075V47h10.458V20.76c1.226-1.882 5.34-2.693 6.97-2.761V8.208a24.963 24.963 0 0 0-2.62-.047zm-64.303.466V0H50.889v41.392s6.1 5.768 14.807 6.074c.865.03 1.742.015 2.62-.047v-9.791c-1.63-.066-5.746-.879-6.97-2.762v-15.75h6.97V8.659h-6.97v-.032z" fill="#090947"/></svg>
```
Please use this optimised SVG

File: entries/i/interia.pl.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "Interia": {
+    "domain": "interia.pl",
+    "url": "https://www.interia.pl/",
```
Comment (ApCoder123) [inline entries/i/interia.pl.json]:
```suggestion
```
The `url` field is not necessary here, as the site is accessible from https://`domain`

File: entries/i/interia.pl.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "contact": {
+      "email": "pomoc@firma.interia.pl",
+      "facebook": "interia"
+    },
+    "keywords": [
+      "email"
+    ]
```
Comment (ApCoder123) [inline entries/i/interia.pl.json:13]:
```suggestion
    ],
    "regions": [
      "pl"
    ]
```

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Review by ApCoder123 — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**